dev/com.ibm.ws.microprofile.reactive.streams.operators_fat_tck/publish/servers/ReactiveStreamsTCKServer/jvm.options

```diff
@@ -1 +1 @@
--Xtrace:iprint=mt,methods={org/apache/maven/surefire/booter/ForkedBooter.main},trigger=method{org/apache/maven/surefire/booter/ForkedBooter.main,jstacktrace},stackdepth=1
+-Dcom.ibm.tools.attach.enable=yes
```

Comment:
I guess we should ask @benjamin-confino what that -Xtrace stuff was for? Doesn't work on openjdk